src/AWSS3.jl

```diff
@@ -179,7 +179,7 @@ function s3_get(
     end
 end
 
-s3_get(a...; b...) = s3_get(global_aws_config(), a...; b...)
+s3_get(bucket, path; kwargs...) = s3_get(global_aws_config(), bucket, path; kwargs...)
```

Comment:
Last thing to deal with: as we're moving from `args...` to specifically support a fixed amount of args this will need to be marked as breaking. Did you want to make this a breaking change?

Reply:
:lol-cry: this was the originating issue: #213. The point in making this switch in the first place was to make it less likely for newer users to get sensical error messages.... so while this seems like a silly issue to make a breaking release over, without making this change then this whole PR was kinda pointless. as a maintainer, lmk which you'd prefer---we can also ditch this PR outright!

Reply:
I should have read through the original issue. My bad. I think there may be a non-breaking alternative here. I'll dig into it

Reply:
Okay, I've cooked up a non-breaking pattern:
```julia
julia> using AWS

julia> foo(::AbstractAWSConfig, bucket, key) = (bucket, key)
foo (generic function with 1 method)

julia> foo(args...) = foo(global_aws_config(), args...)
foo (generic function with 2 methods)

julia> foo("bucket", "key")
("bucket", "key")

julia> foo("bucket", "key", 0)
ERROR: StackOverflowError:

julia> foo(aws::AbstractAWSConfig, args...) = throw(MethodError(foo, Tuple{typeof(aws), typeof(args).types...}))
foo (generic function with 3 methods)

julia> foo("bucket", "key", 0)
ERROR: MethodError: no method matching foo(::AWSConfig, ::String, ::String, ::Int64)
Closest candidates are:
  foo(::AbstractAWSConfig, ::Any...) at REPL[6]:1
  foo(::Any...) at REPL[3]:1
  foo(::AbstractAWSConfig, ::Any, ::Any) at REPL[2]:1
Stacktrace:
 [1] foo(::AWSConfig, ::String, ::String, ::Int64)
   @ Main ./REPL[6]:1
 [2] foo(::String, ::String, ::Int64)
   @ Main ./REPL[3]:1
 [3] top-level scope
   @ REPL[7]:1

julia> foo("bucket", "key")
("bucket", "key")
```